### PR TITLE
Install node packages in one task instead of 3

### PR DESCRIPTION
**Pull request by michaelgugino**
This commit reduces the number of package tasks
from 3 to 1.

### CODE REVIEW

File: roles/openshift_node/tasks/install.yml [hunk 1]
```diff
@@ -1,28 +1,15 @@
...
-
-  - name: Install conntrack-tools package
-    package:
-      name: "conntrack-tools"
-      state: present
-    register: result
-    until: result is succeeded
+- name: Install Node package, sdn-ovs, conntrack packages
+  package:
+    name:
```
Comment (sdodson) [inline roles/openshift_node/tasks/install.yml]: `name: "{{ item }}"` or is this somehow implicit?

Comment (michaelgugino) [reply roles/openshift_node/tasks/install.yml]: Yeah, this patch needs some tweaking.  I'm not sure what I was doing here.

**Comment by sdodson** [comment]
https://github.com/openshift/openshift-ansible/blob/master/roles/openshift_master/tasks/upgrade/rpm_upgrade.yml may be worth considering as it will install all packages in one transaction but that will make it a bit opaque anytime there's an error installing one of the particular packages.

**Comment by michaelgugino** [comment]
> https://github.com/openshift/openshift-ansible/blob/master/roles/openshift_master/tasks/upgrade/rpm_upgrade.yml may be worth considering as it will install all packages in one transaction but that will make it a bit opaque anytime there's an error installing one of the particular packages.

It's not typically necessary to join the packages because the ansible module is smart enough in this case to combine with_items into one transaction (this is a special case for the package/apt/yum modules).

I just missed some things in this patch, I will get a new patch our shortly.

**Pull request merged**